src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -211,7 +211,8 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config, private
                     messages.getMessageWithBotSignature(
                         issue.project.key, config[Modules.FutureVersion.message]
                     )
-                )
+                ),
+                ::resolveAs.partially1(issue).partially1("Invalid")
```

Comment:
I suppose you meant `Awaiting Response`

Reply:
Its a good way to make sure we dont have the problem :P

fixed